src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/PhysicalResourceDetails.js

```diff
@@ -69,23 +69,41 @@ export const PhysicalResourceDetails = ({
     VOLUME: index => getSubfieldName(PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.VOLUMES, 0, index),
   };
 
+  const physicalDetailsDisabledPaths = [
+    PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.MATERIAL_SUPPLIER,
+    PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.CREATE_INVENTORY,
+    PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.RECEIPT_DUE,
+    PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.EXPECTED_RECEIPT_DATE,
+    getFieldName(PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.MATERIAL_TYPE),
+  ];
+
   const { formatMessage } = useIntl();
 
-  const [poStatus] = useFieldMappingFieldValue([PO_STATUS_FIELD]);
-  const [createInventory] = useFieldMappingFieldValueByPath(CREATE_INVENTORY_PHYSICAL_PATH);
+  const [poStatus, orderFormat] = useFieldMappingFieldValue([PO_STATUS_FIELD, ORDER_FORMAT_FILED]);
   const [volumes] = useFieldMappingRefValues([VOLUMES_FIELD]);
   const [materialSupplierId, mappingValue] = useFieldMappingValueFromLookup(MATERIAL_SUPPLIER_FIELD);
 
   const isCreateInventoryDisabled = useMemo(() => poStatus === PO_STATUS.OPEN, [poStatus]);
+  const isPhysicalDetailsDisabled = useMemo(() => orderFormat === ORDER_FORMATS.ELECTRONIC_RESOURCE, [orderFormat]);
 
   useEffect(() => {
-    setFieldValue({
-      path: PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.CREATE_INVENTORY,
+    clearFieldValue({
+      paths: [PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.CREATE_INVENTORY],
       isDisabled: isCreateInventoryDisabled,
-      value: createInventory,
       setReferenceTables,
     });
-  }, [isCreateInventoryDisabled, createInventory]); // eslint-disable-line react-hooks/exhaustive-deps
+    clearFieldValue({
+      paths: physicalDetailsDisabledPaths,
+      isDisabled: isPhysicalDetailsDisabled,
+      setReferenceTables,
+    });
+    clearFieldValue({
+      paths: [`profile.mappingDetails.mappingFields[${PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.VOLUMES}].subfields`],
+      isDisabled: isPhysicalDetailsDisabled,
+      setReferenceTables,
+      value: [],
+    });
+  }, [isCreateInventoryDisabled, isPhysicalDetailsDisabled]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Split this into separate `useEffects` with different dependencies, because every time `isCreateInventoryDisabled` prop has changed but `isPhysicalDetailsDisabled` not, `physicalDetailsDisabledPaths` and `[profile.mappingDetails.mappingFields[${PHYSICAL_RESOURCE_DETAILS_FIELDS_MAP.VOLUMES}].subfields]` will also be changed, it's unnecessary re-rendering